packages/seacas/libraries/ioss/src/exodus/Ioex_DatabaseIO.C

```diff
@@ -1298,6 +1298,10 @@ namespace Ioex {
     m_groupCount[entity_type] = used_blocks;
 
     if (entity_type == EX_ELEM_BLOCK) {
+      if(!assemblyOmissions.empty() || !assemblyInclusions.empty()) {
```

Comment:
Ioex::ParallelDatabaseIO probably needs similar code in `get_blocks`

Reply:
Ok, I'll add that too